### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by Nuga0909**
In this pull request, we carried out :

- unit tests with React Testing Library.
- unit tests with snapshots with Jest.

### CODE REVIEW

File: src/tests/navBar.test.js
Comment (LionRouge1) [inline src/tests/navBar.test.js:1]: - [x] Nice work for testing the the components :clap: However some of the test are not passing. Kindly make sure that all the tests are passing
Screenshot from 2023-09-22 07-22-42

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @Nuga0909, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Write unit tests for the files `operate.js` and `calculate.js` using Jest. :heavy_check_mark: 
- Application still work as it's supposed to :heavy_check_mark: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
- [x] According to the GitHub history two persons contributed to this project but in your README file under the **Authors** section, we have only the information about one person listed. Kindly make sure all the authors for this project are listed under the **Authors** section to make your project look more professional.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -87,6 +87,12 @@ To run the project, execute the following command:
 - Twitter: [@twitterhandle](https://twitter.com/nuga0909)
 - LinkedIn: [LinkedIn](linkedin.com/in/adeshina-adenuga-282036171)
 
+👤 **King Immanuel**
+
+- GitHub: [@Kingno3l]
+- Twitter: [@Kingno3l]
+
```
Comment (danmainah) [inline README.md]: - [x] @Team great work adding all authors in the readme, however, the handles are supposed to be links to the author's social media accounts eg Twitter LinkedIn, and GitHub, kindly consider make the handles working links

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Nuga0909 and @Kingno3l , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️  Implemented the change requested to ensure all tests are passing.

 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @Nuga0909, @Kingno3l, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👏 You did a great job adding social media links to the authors section as requested by the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**